### PR TITLE
[ADD] double booking confirmation prompt

### PR DESCRIPTION
**Pull request by lgharib**
Add double booking confirmation prompt

### CODE REVIEW

File: project_resource_calendar/templates/assets.xml [hunk 1]
```diff
@@ -8,9 +8,13 @@
     </template>
     <template id="assets_backend" name="update event wizard" inherit_id="web.assets_backend">
         <xpath expr="//script[last()]" position="after">
-            <script type="text/javascript" src="/project_resource_calendar/static/src/js/update_event_wizard.js"/>
             <link href="/project_resource_calendar/static/src/css/calendar_style.css"
                   rel="stylesheet"/>
+            <script type="text/javascript" src="/project_resource_calendar/static/src/js/update_event_wizard.js"/>
+            <script type="text/javascript" src="/project_resource_calendar/static/src/js/save_event_wizard.js"/>
+        </xpath>
+        <xpath expr="//script[@src='/web/static/src/js/views/view_dialogs.js']" position="replace">
```
Comment (rimbendhaou) [inline project_resource_calendar/templates/assets.xml:16]: I think you have to add the web module to the depends in the project_resource_calendar manifest

File: project_resource_calendar/i18n/fr.po [hunk 2]
```diff
@@ -769,39 +798,43 @@ msgstr ""
...
-#: code:addons/project_resource_calendar/models/calendar_event.py:199
+#: code:addons/project_resource_calendar/models/calendar_event.py:232
 #, python-format
-msgid "The number of repetitions must be greater than 0"
-msgstr "Le nombre de répétitions doit être supérieur à zero"
+msgid ""
+"The following resources can not be double-booked:\n"
+" %s"
+msgstr ""
+"Les ressources suivantes de peuvent pas être doublement réservées:\n"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po]:
```suggestion
"Les ressources suivantes ne peuvent pas être doublement réservées:\n"
```

File: project_resource_calendar/i18n/fr.po [hunk 3]
```diff
@@ -238,25 +248,40 @@ msgid "Description"
...
-msgstr "Validation de la double réservation"
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:172
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:59
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:114
+#: code:addons/project_resource_calendar/static/src/js/update_event_wizard.js:47
+#: code:addons/project_resource_calendar/static/src/js/update_event_wizard.js:86
+#, python-format
+msgid "Do you want to continue?"
+msgstr "Voulez vous continuer ?"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:276]:
```suggestion
msgstr "Voulez vous continuer?"
```

File: project_resource_calendar/i18n/fr.po [hunk 4]
```diff
@@ -238,25 +248,40 @@ msgid "Description"
...
 #. module: project_resource_calendar
-#: model:ir.model.fields,field_description:project_resource_calendar.field_calendar_event_double_bookable_equipment_ids
-msgid "Double bookable equipments"
-msgstr "Équipement double réservable"
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:140
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:27
+#, python-format
+msgid "Duration"
+msgstr "Duration"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po]:
```suggestion
msgstr "Durée"
```

File: project_resource_calendar/i18n/fr.po [hunk 5]
```diff
@@ -274,9 +299,11 @@ msgid "End Date"
...
 
 #. module: project_resource_calendar
-#: model:ir.model.fields,field_description:project_resource_calendar.field_doublebooking_validation_wiz_end_date
-msgid "Ending Date"
-msgstr "Date de fin"
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:90
+#, python-format
+msgid "Ending date > Starting date"
+msgstr "Ending date > Starting date"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po]:
```suggestion
msgstr "Date de fin > Date de début"
```

File: project_resource_calendar/i18n/fr.po [hunk 6]
```diff
@@ -707,6 +713,27 @@ msgstr "SKU"
 msgid "Saturday"
 msgstr "Samedi"
 
+#. module: project_resource_calendar
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:125
+#, python-format
+msgid "Save"
+msgstr "Save"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:722]:
```suggestion
msgstr "Sauvegarder"
```

File: project_resource_calendar/i18n/fr.po [hunk 7]
```diff
@@ -939,48 +972,3 @@ msgstr "Rapport d'événement hebdomadaire"
 #: model:ir.model,name:project_resource_calendar.model_weekly_report_wizard
 msgid "weekly.report.wizard"
 msgstr "weekly.report.wizard"
-
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:943]: Dont delete the following translations. The following translations are added manually. that's why Poedit delete them each time but we must keep them.

File: project_resource_calendar/i18n/fr.po [hunk 8]
```diff
@@ -1,58 +1,50 @@
...
+"Plural-Forms: \n"
+"Language: fr_CA\n"
+"X-Generator: Poedit 2.2.3\n"
+
+#. module: project_resource_calendar
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:125
+#, python-format
+msgid " & Close"
+msgstr " & Close"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:25]: why not translated?

File: project_resource_calendar/i18n/fr.po [hunk 9]
```diff
@@ -172,6 +159,13 @@ msgstr "Client"
 msgid "Client Id Partner Ids Names"
 msgstr "ID/Nom du client/partenaire"
 
+#. module: project_resource_calendar
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:111
+#, python-format
+msgid "Close"
+msgstr "Close"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:167]: why not translated

File: project_resource_calendar/i18n/fr.po [hunk 10]
```diff
@@ -193,7 +187,24 @@ msgid "Contact"
...
+#, python-format
+msgid "Continue"
+msgstr "Continuer"
+
+#. module: project_resource_calendar
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:431
+#, python-format
+msgid "Create"
+msgstr "Create"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:205]: why not translated

File: project_resource_calendar/i18n/fr.po [hunk 11]
```diff
@@ -238,25 +248,40 @@ msgid "Description"
 msgstr "Description"
 
 #. module: project_resource_calendar
-#: model:ir.model.fields,field_description:project_resource_calendar.field_doublebooking_validation_wiz_display_name
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:111
+#, python-format
+msgid "Discard"
+msgstr "Discard"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:255]: why not translated

File: project_resource_calendar/i18n/fr.po [hunk 12]
```diff
@@ -707,6 +713,27 @@ msgstr "SKU"
...
+#, python-format
+msgid "Save"
+msgstr "Save"
+
+#. module: project_resource_calendar
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:198
+#, python-format
+msgid "Save & New"
+msgstr "Save & New"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:729]: translation?

File: project_resource_calendar/i18n/fr.po [hunk 13]
```diff
@@ -707,6 +713,27 @@ msgstr "SKU"
...
+
+#. module: project_resource_calendar
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:438
+#, python-format
+msgid "Select"
+msgstr "Select"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:736]: translation?

File: project_resource_calendar/i18n/fr.po [hunk 14]
```diff
@@ -729,9 +756,14 @@ msgid "Start Date"
...
-msgid "Starting Date"
-msgstr "Date de début"
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:136
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:23
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:82
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:86
+#, python-format
+msgid "Starting at"
+msgstr "Starting at"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:767]: transaltion?

File: project_resource_calendar/i18n/fr.po [hunk 15]
```diff
@@ -745,17 +777,14 @@ msgstr "Dimanche"
...
 #. module: project_resource_calendar
-#: code:addons/project_resource_calendar/models/calendar_event.py:250
+#: code:addons/project_resource_calendar/models/calendar_event.py:244
 #, python-format
-msgid ""
-"The attendee %s cannot be double-booked with any overlapping meetings or "
-"events."
-msgstr "Le participant, %s, est déjà réservé pour la période spécifiée."
+msgid "The %s: %s From:%s To:%s\n"
+msgstr "The %s: %s From:%s To:%s\n"
```
Comment (rimbendhaou) [inline project_resource_calendar/i18n/fr.po:788]: translation?

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_resource_calendar/templates/assets.xml (hunk 1 above)
Comment (lgharib) [reply project_resource_calendar/templates/assets.xml:16]: Yes

File: project_resource_calendar/i18n/fr.po (hunk 11 above)
Comment (lgharib) [reply project_resource_calendar/i18n/fr.po:255]: I wasn't originally in the POT file I downloaded from Odoo to add new translations.

File: project_resource_calendar/i18n/fr.po (hunk 12 above)
Comment (lgharib) [reply project_resource_calendar/i18n/fr.po:729]: I wasn't originally in the POT file I downloaded from Odoo to add new translations.

File: project_resource_calendar/i18n/fr.po (hunk 15 above)
Comment (lgharib) [reply project_resource_calendar/i18n/fr.po:788]: I wasn't originally in the POT file I downloaded from Odoo to add new translations.

File: project_resource_calendar/i18n/fr.po (hunk 14 above)
Comment (lgharib) [reply project_resource_calendar/i18n/fr.po:767]: I wasn't originally in the POT file I downloaded from Odoo to add new translations.

File: project_resource_calendar/i18n/fr.po (hunk 3 above)
Comment (lgharib) [reply project_resource_calendar/i18n/fr.po:276]: A space is needed before and after the question mark in French.

**Pull request closed without merging**